### DISCUSSION TITLE
Client application and Signoz on different Windows server

### DISCUSSION THREAD

**Discussion by tommy38hk** [SigNoz/signoz#1199] [Q&A]
When Signoz is running locally via Docker with the client application in the same server, everything is working. However, if I configure the same application on another Windows server, I got connection timed out exception. There's no firewall issue as if I host another dummy application on port 4317, the other server can access the resource. 

Please help

**Comment by pranay01** [comment]
@tommy38hk have you tried the troubleshooting instructions - https://signoz.io/docs/install/troubleshooting/

Might be helpful in figuring out the issue. Though we don't officially support running SigNoz on windows servers as we have not tested on Windows servers. If you have linux servers, might be best to run SigNoz on it

**Comment by tommy38hk** [comment]
I cannot run the troubleshooting tool as the application is hosting on Windows. I just have Signoz running locally (on another Windows) using docker.
I thought client application can be on Windows, right?

**Reply by pranay01** [reply]
Yes, client application on Windows should work, but we have not tested this. So there might be some nuances  here. Can you check opentelemetry community on this, as we support opentelemetry instrumentation
